src/Resources/ShipmentOptions.php

```diff
@@ -61,6 +64,24 @@ public function setDescriptionAttribute(string $value): void
         $this->label_description = $value;
     }
 
+    /**
+     * Set Insurance amount. Only applicable for package_type = 1 (package)
+     * @param  int  $value
+     * @param  string  $currency
+     * @return $this
+     */
+    public function setInsurance(int $value, string $currency = 'EUR'): self
```

Comment:
This method should be `setInsuranceAttribute`.

Reply:
Why should it be setInsuranceAttribute? is that for bw compat with Laravel ?

Reply:
It is a magic setter, just like the way Laravel does it.

Reply:
We should make the `insurance` attribute protected I guess. So you would always need to go through this setter.